R/simulate.R

```diff
@@ -61,15 +60,15 @@ spim_simulate_prepare <- function(combined, simulate_parameters, n_par,
     pars_mcmc, combined$transform)
 
   if (inflate_strain) {
-    message("Inflating strains")
+    message("Inflating strains. NOT SURE IF THIS WORKS WITH MULTISTAGE FITTING")
```

Comment:
No need for this comment - we should remove both inflations; I've added this issue for this: https://github.com/mrc-ide/spimalot/issues/153

```suggestion
    message("Inflating strains")
```

Reply:
done (just removed from this function)